misaligned/misaligned.cs

```diff
@@ -16,7 +16,7 @@ class Misaligned {
         }
         static void Main(string[] args) {
             int result = printColorMap();
-            Debug.Assert(result == 25);
+            Debug.Assert(result == 24);

```

Comment:
The number of pairs is a product of number of major colors and number of minor colors. Right now the value is 25. Do you have a different logic to arrive at 24?